Gemfile

```diff
@@ -12,7 +12,8 @@ gemspec
 # This is needed if the version of the gem you want to use is not on rubygems
 
 LOCAL_DEV = false
-MINIMAL_GEMS = false    # Keep only one non-native gem, and one native
+MINIMAL_GEMS = false   # Keep only one non-native gem, and one native
+FINAL_PACKAGE = !ENV['FINAL_PACKAGE'].nil?
```

Comment:
THe first time bundle is run (for caching purposes), this is false.

When I install from the local cache, this is true